File: Data Structures & Algorithms/permutations/submission-0.py
class Solution:
    def permute(self, nums: List[int]) -> List[List[int]]:
        output=[]
        def backtrack(path):
            if(len(path)==len(nums)):
                output.append(path[:])
            for num in nums:
                if num in path:
                    continue
                path.append(num)
                backtrack(path)
                path.pop()
        backtrack([])
        return output

File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-0.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> heap= new PriorityQueue<>((a,b)->b-a);
        for(int num:nums){
            heap.add(num);
        }
        int output=0;
        while(k!=0){
            k--;
            output=heap.poll();
        }
        return output;
    }
}
